### ISSUE TITLE
[A11y]Incorrect name property provided for the "Custom" tab item.

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#7884]
Copied from https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1069506
Steps to Reproduce:
1. Hit the URL:https://www.nuget.org/
2. Navigate to "Sign in" control and hit enter.
3. Navigate to "Sign in with Microsoft" control and hit enter.
4. Select “Upload Package” from the Profile Dropdown and hit enter.
5. Navigate to "Custom" control and hit enter.
6. Open AI4D.
7. Check name property for the "Cutom" tab item.

Actual:
For the "Custom" tab item, Incorrect name property provided. 

Expected:
For the "Custom" tab item, Name property should provided correctly and screen reader should announces the same.

Notes:
Looks like we set the name of this to "Enter readme in Markdown"
We can probably either change this to be "Custom Markdown" or "Custom" assuming that the context of this is sufficient. I lean towards not fix since this is more descriptive from a usability perspective.

**Comment by ryuyu** [comment]
Update name to read "Label : Description." This makes the tab names clear.
